index.js

```diff
@@ -4,7 +4,40 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+   //console.log(process.argv)
+    
+   if(!process.argv[3]) {
+        
+    if(!process.argv[2]) {
+        return "No operation provided...";
+}
```

Comment:
Be careful with your formatting. This bracket makes it seem like this is the end of your function. Make sure to have the proper indentation.